docs/en/metrics/metrics-overview.asciidoc

```diff
@@ -19,7 +19,7 @@ Metrics monitoring requires the following {stack} components.
 distributed storage, search, and analytics engine. {es} can store, search, and analyze large volumes of data in near real-time.
 The {metrics-app} uses {es} to store metrics data in {es} documents which are queried on demand.
 
-*https://www.elastic.co/products/beats[{beats}]* are open source data shippers that you install as agents on your servers to send data to {es}.
+*{metricbeat-ref}[{metricbeat}]* is an open source data shipper that you install as an agent on your servers to send data to {es}.
```

Comment:
I think you might want to revert this change. `{metricbeat-ref}` resolves the path to the Metricbeat reference. But here, I think you want to point to the product page like you do for ES and Kibana.

Reply:
Will do. Thanks.